source/settings/telemetry.txt

```diff
@@ -0,0 +1,132 @@
+.. _compass-telemetry:
+
+===========================
+Configure Telemetry Options
+===========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+|compass-short| collects anonymous aggregated usage data to improve
+MongoDB products. |compass-short| collects this information by default,
+but you can disable this data collection.
+
+About this Task
+---------------
+
+|compass-short| tracks the following data:
+
+- The type of MongoDB deployment |compass-short| is connected to. For
+  example, Enterprise Edition, Community Edition, or Atlas Data Lake.
+
+- User actions performed in |compass-short|, such as creating
+  collections and indexes. |compass-short| only tracks the general
+  action performed, and does not track field values or namespaces.
+
+- Errors.
+
+|compass-short| *does not* track:
+
+- IP addresses, hostnames, usernames, or credentials.
+
+- Queries run in |compass-short|.
+
+- Data stored in your MongoDB deployment.
+
+- Personal identifiable information.
+
+For more information, see `MongoDB's Privacy Policy
+<https://www.mongodb.com/legal/privacy-policy?tck=docs_mongosh>`__.
+
+.. tip::
+
+   |compass-short| Isolated Edition automatically disables telemetry.
+   For more information on different |compass-short| editions, see
+   :ref:`compass-feature-table`.
+
+Steps
+-----
+
+You can toggle telemetry in either: 
+
+- The |compass-short| :ref:`Settings panel <compass-settings-reference>`
+
+- :ref:`Command line options <cli-options>`
+
+- :ref:`Configuration file options <config-file>`
+
+Settings Panel
+~~~~~~~~~~~~~~
+
+To toggle telemetry from the |compass-short| settings:
+
+.. procedure:: 
+   :style: normal
+   
+   .. step:: Open the Settings window
+      
+      a. In the |compass-short| top menu bar, click :guilabel:`MongoDB Compass`
+      
+      b. Click :guilabel:`Settings`
+      
+         .. figure:: /images/compass/settings-panel-location.png
+            :scale: 40% 
+            :alt: Settings panel location under the MongoDB Compass system menu
+
+      .. tip::
+
+         Alternatively, you can use keyboard shortcuts to open the 
+         :guilabel:`Settings` panel: 
+         
+         - Windows / Linux: ``Ctrl`` + ``,``
+         
+         - Mac: ``⌘`` + ``,``
+
+   .. step:: In the :guilabel:`Settings` window, click :guilabel:`Privacy`
+   
```

Comment:
[question] do you think it would be helpful to add a screenshot of the window here?

Reply:
My preference would be to omit the screenshot to help future-proof this in the event that the UI changes. I added a bit further clarification to say that "Privacy" is in the left pane of the window, which should hopefully ensure that users can find it.